chapters/8-annotations.md

```diff
@@ -132,16 +132,16 @@ SPDXREF: DocumentRef-spdx-tool-1.2:SPDXRef-5
 
 **8.4.6** RDF:
 
-For RDF, the annotations are a property of the SPDX element it is annotationg.
+For RDF, the annotations are a property of the SPDX Document, Package, File, or Snippet they are annotationg.
```

Comment:
minor nit (in original, not from your edit) -- "annotationg" => "annotating"

Reply:
done